### PR TITLE
Update Build and Run

### PR DESCRIPTION
**Pull request by wrighted**
Updates the README to include helpful instructions for running this project.

Reviewers should ensure they can run this project, to ensure there are no steps missing.

### CODE REVIEW

**Comment by wrighted** [comment]
Update 2 rebases to `main` to allow checks to pass.

File: README.md [hunk 1]
```diff
@@ -1,3 +1,65 @@
...
+     cd MauiFretty
+     dotnet restore Fretty.sln
+
+Next build and publish the solution:
+     
+     mkdir artifacts
+     dotnet build -f <target> -c Release -o ./artifacts
+     dotnet publish -f <target> -c Release -p:BuildIpa=True -o ./artifacts
+
+where `<target>` is the intended target (MacOS, iOS, Windows, Android).
```
Comment (alyssa-blair) [inline README.md]: maybe update this line to be less ambiguous

File: README.md [hunk 2]
```diff
@@ -1,3 +1,65 @@
 # Fretty in Maui
 
-Yet another guitar assistant with no build instructions. :)
+Fretty is a helpful guitar assistant implemented in .NET and C++. Time to get ready for Fretty!
```
Comment (alyssa-blair) [inline README.md:3]: what happened to c# :(

Comment (wrighted) [reply README.md:3]: C# is .NET :)

File: README.md (hunk 1 above)
Comment (wrighted) [reply README.md]: See update 2.

**Review by alyssa-blair — APPROVED** [review]
ya good enough

**Pull request merged**